tailscale/config.yaml

```diff
@@ -25,3 +31,7 @@ devices:
 host_network: true
 schema:
   tags: ["match(^tag:[a-zA-Z0-9]-?[a-zA-Z0-9]+$)?"]
+  request_cert: bool
+  domain_alias: match(^[a-z0-9][a-z0-9\-_]{0,61}\.ts\.net$)?
```

Comment:
Maybe we don't need `request_cert` at all? For example, if one provides the domain alias, that should already be enough?

Reply:
And if we want to temporarily stop renewal?

Reply:
That is a shallow use case to justify it imho. If wanted, one could temporary remove the alias as well 🤷

Reply:
Yeah makes sense. We can do without request_cert.

Reply:
@frenck yes, it is a shallow use case, I'm fine with removing the `request_cert`. My "concern" was that if someone uses a complex regex, he would have to save it somewhere. But anyway, still shallow :-)

Reply:
Users don't input regex though. The regex validates the config option which expects a domain name in the form xyz.ts.net where xyz is your tailnet domain alias from the tailscale admin.